catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/UdpStreamMonitor.java

```diff
@@ -523,15 +523,15 @@ public Integer getByteCountRolloverCondition() {
     /**
      * @param count must be non-null and positive
      */
-    public void setByteCountRolloverCondition(Integer count) {
+    public void setMegabyteCountRolloverCondition(Integer count) {
         notNull(count, "count must be non-null");
 
         inclusiveBetween(BYTE_COUNT_MIN,
```

Comment:
I changed the limits to use the min and max values defined in MegabyteRolloverCondition